packages/react-core/src/components/DescriptionList/DescriptionListDescription.tsx

```diff
@@ -0,0 +1,21 @@
+import * as React from 'react';
+import styles from '@patternfly/react-styles/css/components/DescriptionList/description-list';
+import { css } from '@patternfly/react-styles';
+
+export interface DescriptionListDescriptionProps extends React.HTMLProps<HTMLElement> {
+  /** Anything that can be rendered inside of list description */
+  children: React.ReactNode;
+  /** Additional classes added to the DescriptionListDescription */
+  className?: string;
+}
+
+export const DescriptionListDescription: React.FunctionComponent<DescriptionListDescriptionProps> = ({
+  children = null,
+  className,
+  ...props
+}: DescriptionListDescriptionProps) => (
+  <dd className={css(styles.descriptionListDescription, className)} {...props}>
+    <div className={'pf-c-description-list__text'}>{children}</div>
```

Comment:
is this class defined on `styles.descriptionListText`?

Reply:
No it isn't @jschuler - there is no `.pf-c-description-list__text` class or corresponding styles in https://github.com/patternfly/patternfly/blob/master/src/patternfly/components/DescriptionList/description-list.scss, but this class is still applied in the core examples for some reason